README.md

```diff
@@ -1 +1,70 @@
-# bookstore
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
```

Comment:
- [ ] I would recommend removing the auto generated README from `npx create-react app` and use the [Microverse README template](https://github.com/microverseinc/readme-template). It should includes project title, description, author detail, steps to clone and run the project locally, etc. to make the README professional.